Logos/LastPass.svg

```diff
@@ -0,0 +1 @@
45c0-.4-.22-.756-.63-1.008s-1.008-.504-1.795-.787L83.456 47c-.44-.22-.82-.44-1.134-.724a2.61 2.61 0 0 1-.693-.882c-.157-.346-.252-.724-.252-1.165 0-.472.063-.913.252-1.323s.44-.756.787-1.04.787-.535 1.3-.724 1.102-.283 1.795-.283a12.28 12.28 0 0 1 1.858.126c.63.094 1.228.252 1.795.472.126.063.2.126.22.22a.56.56 0 0 1 0 .315l-.472 1.26c-.03.126-.094.2-.157.2-.063.03-.157 0-.283-.03-.567-.2-1.07-.315-1.512-.378a10.04 10.04 0 0 0-1.417-.094c-.535 0-.945.126-1.197.346s-.378.472-.378.756c0 .2.03.378.126.535a1.29 1.29 0 0 0 .41.41 4.57 4.57 0 0 0 .693.378 16.16 16.16 0 0 0 1.071.409l1.386.535c.44.2.85.4 1.197.693a3.56 3.56 0 0 1 .85.976c.22.378.346.82.346 1.386.094.535 0 1.04-.22 1.512zm10.203-1.544c0 .598-.094 1.102-.315 1.543s-.504.82-.913 1.102c-.378.283-.85.504-1.386.66s-1.102.22-1.732.252c-.346 0-.724 0-1.134-.03s-.787-.063-1.197-.157a6.98 6.98 0 0 1-1.165-.283c-.378-.126-.724-.252-1.04-.44-.22-.126-.283-.283-.157-.535l.504-1.228a.6.6 0 0 1 .22-.252c.063-.032.2-.03.346.063.535.252 1.134.4 1.763.535.66.126 1.228.2 1.732.2.63 0 1.07-.126 1.386-.378s.44-.567.44-.945c0-.4-.22-.756-.63-1.008s-1.008-.504-1.795-.787l-1.606-.63c-.44-.22-.82-.44-1.134-.724a2.61 2.61 0 0 1-.693-.882c-.157-.346-.252-.724-.252-1.165 0-.472.063-.913.252-1.323.157-.4.44-.756.787-1.04s.787-.535 1.3-.724 1.102-.283 1.795-.283a12.28 12.28 0 0 1 1.858.126c.63.094 1.228.252 1.795.472.126.063.2.126.22.22a.56.56 0 0 1 0 .315l-.472 1.26c-.03.126-.094.2-.157.2-.063.03-.157 0-.283-.03-.567-.2-1.07-.315-1.512-.378a10.04 10.04 0 0 0-1.417-.094c-.535 0-.945.126-1.197.346s-.378.472-.378.756c0 .2.03.378.126.535a1.29 1.29 0 0 0 .41.41 4.57 4.57 0 0 0 .693.378 16.16 16.16 0 0 0 1.071.409l1.386.535c.44.2.85.4 1.197.693a3.56 3.56 0 0 1 .85.976c.346.346.44.82.44 1.354" fill="#d32d27"/><path d="M40.067 51.247c0 .346-.126.598-.41.724a5.06 5.06 0 0 1-1.039.378l-1.165.252-1.197.126c-.378.03-.756.03-1.102.03-.976 0-1.795-.094-2.425-.315-.63-.19-1.134-.472-1.512-.787-.378-.346-.63-.724-.787-1.197-.157-.44-.22-.945-.22-1.45 0-.535.094-1.008.252-1.512.157-.472.44-.913.82-1.26.378-.378.882-.66 1.48-.882s1.354-.346 2.236-.346h2.488v-.378c0-.283-.03-.535-.094-.787s-.19-.44-.346-.63-.41-.315-.693-.41-.693-.157-1.134-.157c-.504 0-1.04.03-1.637.126s-1.07.19-1.45.315c-.157.03-.283.063-.346.03s-.126-.126-.157-.252l-.315-1.26c-.063-.252 0-.378.19-.472.63-.252 1.29-.41 2.047-.504.724-.094 1.417-.157 2.078-.157 1.543 0 2.645.346 3.37 1.04s1.07 1.67 1.07 2.992zm-2.614-4.188h-2.1l-.82.03c-.283.03-.567.126-.82.252s-.472.315-.63.567-.252.598-.252 1.04c0 .63.22 1.07.63 1.323.44.252 1.008.378 1.7.378h.44c.19 0 .378-.03.598-.063l.66-.126a2.09 2.09 0 0 0 .63-.189V47.06z"/><g transform="matrix(1.862268 0 0 1.862268 -132.02296 -35.415701)"><path d="M115.106 23.546a.71.71 0 1 1 1.449 0v11.588a.71.71 0 1 1-1.449 0z" fill="#d32d27"/><g transform="matrix(.314897 0 0 .314897 89.670313 27.163913)" fill="#d32d27"><circle r="7.8" cy="7.8" cx="7.8"/><circle r="7.8" cy="7.8" cx="33.8"/><circle r="7.8" cy="7.8" cx="59.8"/></g></g></svg>
```

Comment:
As per [guidelines ](https://microsoft.sharepoint.com/:x:/r/teams/AzureSecurityInsights/_layouts/15/guestaccess.aspx?e=aETsaQ&share=EWs_HuWia89Ns0pc1jXRWQkB7aNXX6Zi1REQcKtH4gQp5Q&isSPOFile=1)logo should be less than 5 kb. Please update.